module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationKeyCredential.md

```diff
@@ -8,62 +19,103 @@ schema: 2.0.0
 ```
-PS C:\> $AppID = (Get-EntraBetaApplication -Top 1).Objectid
-PS C:\> New-EntraBetaApplicationKeyCredential -ObjectId $AppId -CustomKeyIdentifier "Test" -StartDate "11/7/2016" -Type "Symmetric" -Usage "Sign" -Value "123"
 
+```Output
 CustomKeyIdentifier : {84, 101, 115, 116}
 EndDate             : 11/7/2017 12:00:00 AM
-KeyId               : a5845538-3f67-402d-a03e-36d768f1441e
+KeyId               : aaaaaaaa-0b0b-1c1c-2d2d-333333333333
 StartDate           : 11/7/2016 12:00:00 AM
 Type                : Symmetric
 Usage               : Sign
 Value               : {49, 50, 51}
 ```
 
-The first command gets the ID of an application by using the Get-EntraBetaApplication (./Get-EntraBetaApplication.md)cmdlet.
-The command stores it in the $AppId variable.
+This example shows how to create an application key credential.
 
-The second command creates the application key credential for the application identified by $AppId.
+You can use the `Get-EntraApplication` cmdlet to retrieve the application Object ID.
 
 ### Example 2: Use a certificate to add an application key credential
-```
-PS C:\> $cer = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2 #create a new certificate object
-PS C:\> $cer.Import("C:\Users\PFuller\Desktop\abc.cer") 
-PS C:\> $bin = $cer.GetRawCertData()
-PS C:\> $base64Value = [System.Convert]::ToBase64String($bin)
-PS C:\> $bin = $cer.GetCertHash()
-PS C:\> $base64Thumbprint = [System.Convert]::ToBase64String($bin)
-PS C:\> $keyid = [System.Guid]::NewGuid().ToString() 
-PS C:\> New-EntraBetaApplicationKeyCredential -ObjectId 009d786a-3503-4217-b8ab-db03d71c179a -CustomKeyIdentifier  $base64Thumbprint  -Type AsymmetricX509Cert -Usage Verify -Value $base64Value  -StartDate $cer.GetEffectiveDateString() -EndDate cer.GetExpirationDateString()
-```
 
-The first seven commands create values for the application key credential and stores them in variables.
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+
+$cer = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2 #create a new certificate object
+$cer.Import('C:\Users\PFuller\Desktop\abc.cer') 
+$bin = $cer.GetRawCertData()
+$base64Value = [System.Convert]::ToBase64String($bin)
+$bin = $cer.GetCertHash()
+$base64Thumbprint = [System.Convert]::ToBase64String($bin)
+$keyid = [System.Guid]::NewGuid().ToString() 
+
+$params = @{
+    ObjectId = '22223333-cccc-4444-dddd-5555eeee6666'
+    CustomKeyIdentifier = $base64Thumbprint
+    Type = 'AsymmetricX509Cert'
+    Usage = 'Verify'
+    Value = $base64Value
+    StartDate = $cer.GetEffectiveDateString()
+    EndDate = $cer.GetExpirationDateString()
+}
+
+New-EntraBetaApplicationKeyCredential @params
+```
 
-The final command uses a certificate to add an application key credential.
+This example shows how to create an application key credential.
```

Comment:
In example section we need to add decription for Prarameters used in cmdlets ObjectId, CustomKeyIdentifier
StartDate . Type, Usage , Value,EndDate